### ISSUE TITLE
Error when adding a URL

### ISSUE THREAD

**Issue by homelabtest** [mariosemes/PornHub-downloader-python#18]
Hello,
the new version from today seems to have an error now when adding a pornstar URL, see attachment. The URL is not added to database.
[phdl.txt](https://github.com/mariosemes/PornHub-downloader-python/files/4698341/phdl.txt)

**Comment by sirn7** [comment]
I have this identical error, have you found a resolution?

**Comment by homelabtest** [comment]
No, it seems to be a bug and the fact, that you have the same error, reassures me that I did everything right. I think the author didn't test after the latest release or the error occurs only in a special constellation (that we both have).

**Comment by simba87** [comment]
Created pull request to fix this issue. Feel free to check it with my fork.

**Comment by homelabtest** [comment]
Seems it is possible to start the program again, but the pornstar category still doesn't work (collected 0 videos for https://de.pornhub.com/pornstar/stacy-cruz).

I am new to Linux, so I tell you, what i did:
1) I created a new container and followed the standard installation.
2) I replaced the files requirements.txt and functions.py with your new files.
3) Then I executed 
apt-get install libxml2-dev libxslt-dev python-dev
pip3 install -r requirements.txt
4) After that:
python3 phdler.py add https://de.pornhub.com/pornstar/stacy-cruz
python3 phdler.py start
5) Output:

root@d5166c7e30a6:/PornHub-downloader-python-master# python3 phdler.py start
Running startup checks...
Database exists.
downloading new items

pornstar
stacy-cruz
https://www.pornhub.com/pornstar/stacy-cruz

[PornHubPagedVideoList] pornstar/stacy-cruz/videos: Downloading page 1
[download] Downloading playlist: pornstar/stacy-cruz/videos
[PornHubPagedVideoList] playlist pornstar/stacy-cruz/videos: Collected 0 video ids (downloading 0 of them)
[download] Finished downloading playlist: pornstar/stacy-cruz/videos
downloading all items

pornstar
stacy-cruz
https://www.pornhub.com/pornstar/stacy-cruz

[PornHubPagedVideoList] pornstar/stacy-cruz/videos: Downloading page 1
[download] Downloading playlist: pornstar/stacy-cruz/videos
[PornHubPagedVideoList] playlist pornstar/stacy-cruz/videos: Collected 0 video ids (downloading 0 of them)
[download] Finished downloading playlist: pornstar/stacy-cruz/videos
root@d5166c7e30a6:/PornHub-downloader-python-master#

**Comment by simba87** [comment]
The same issue with Stacy Cruz for me) Looks like she has some specific profile page. For example application works perfectly with other pornstar: https://rt.pornhub.com/pornstar/mila-azul
Look at this output where I am trying to download Stacy and then Mila:

$ python3 phdler.py add https://rt.pornhub.com/pornstar/stacy-cruz
Running startup checks...
Database exists.
PornHub url validated.
This is a PORNSTAR url,
and the URL is existing.
Stacy Cruz added to database.
$ sudo python3 phdler.py start
Running startup checks...
Database exists.
downloading new items
-----------------------------
pornstar
stacy-cruz
https://www.pornhub.com/pornstar/stacy-cruz
-----------------------------
Download URL:https://www.pornhub.com/pornstar/stacy-cruz
[PornHubPagedVideoList] pornstar/stacy-cruz/videos: Downloading page 1
[download] Downloading playlist: pornstar/stacy-cruz/videos
[PornHubPagedVideoList] playlist pornstar/stacy-cruz/videos: Collected 0 video ids (downloading 0 of them)
[download] Finished downloading playlist: pornstar/stacy-cruz/videos
downloading all items
-----------------------------
pornstar
stacy-cruz
https://www.pornhub.com/pornstar/stacy-cruz
-----------------------------
[PornHubPagedVideoList] pornstar/stacy-cruz/videos: Downloading page 1
[download] Downloading playlist: pornstar/stacy-cruz/videos
[PornHubPagedVideoList] playlist pornstar/stacy-cruz/videos: Collected 0 video ids (downloading 0 of them)
[download] Finished downloading playlist: pornstar/stacy-cruz/videos
$ python3 phdler.py add https://rt.pornhub.com/pornstar/mila-azul
Running startup checks...
Database exists.
PornHub url validated.
This is a PORNSTAR url,
and the URL is existing.
Mila Azul added to database.
$ sudo python3 phdler.py start
Running startup checks...
Database exists.
downloading new items
-----------------------------
pornstar
mila-azul
https://www.pornhub.com/pornstar/mila-azul
-----------------------------
Download URL:https://www.pornhub.com/pornstar/mila-azul
[PornHubPagedVideoList] pornstar/mila-azul/videos: Downloading page 1
[PornHubPagedVideoList] pornstar/mila-azul/videos: Downloading page 2
[PornHubPagedVideoList] pornstar/mila-azul/videos: Downloading page 3
[PornHubPagedVideoList] pornstar/mila-azul/videos: Downloading page 4
[PornHubPagedVideoList] pornstar/mila-azul/videos: Downloading page 5
[download] Downloading playlist: pornstar/mila-azul/videos
[PornHubPagedVideoList] playlist pornstar/mila-azul/videos: Collected 191 video ids (downloading 191 of them)
[download] Downloading video 1 of 191
[PornHub] ph5e81b33f86593: Downloading pc webpage
[PornHub] ph5e81b33f86593: Downloading m3u8 information
[PornHub] ph5e81b33f86593: Downloading m3u8 information
[PornHub] ph5e81b33f86593: Downloading m3u8 information
[PornHub] ph5e81b33f86593: Downloading m3u8 information
[download] Destination: /mnt/x/phmedia/pornstar/Mila Azul/Hot Babe Mila Azul in slow motion with perfect natural tits for Nudex.mp4
[download]  62.2% of 135.05MiB at 595.90KiB/s ETA 01:27

**So I guess this is a new issue - you can create it. It seems application has a problem with some type of pages. Please compare Stacy's and Mila's pages - they even look differently.**

**Comment by homelabtest** [comment]
Hmm, bad luck, Stacy is hotter than Mia. ;-)
Thanks for the analysis.

**Comment by homelabtest** [comment]
So, I added a new issue for this. I am new to Github - I suppose I have to close the issue as your bugfix allows starting the program again?

**Comment by simba87** [comment]
I guess this issue can be closed only after my pull request will be merged. But it depends on this repository owner - I have no such permissions.

**Comment by FearNaBoinne** [comment]
Looks like this is abandoned, as even a complete fix pull request isn't pulled in again...

BTW, downloading from Simba87's fork works, and it does work... (Just wish there was a way to do more filtering... If a "model" has ~3000 video's, I probably do not want to download all of them... And with PH deleting playlists without obvious reasons, that may be an issue...  ;) )

**Comment by mariosemes** [comment]
The development of this tools is deprecated.
Feel free to fork it and keep on developing it.
